paimon-core/src/main/java/org/apache/paimon/metrics/commit/CommitMetrics.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.paimon.metrics.commit;
+
+import org.apache.paimon.annotation.VisibleForTesting;
+import org.apache.paimon.metrics.AbstractMetricGroup;
+import org.apache.paimon.metrics.DescriptiveStatisticsHistogram;
+import org.apache.paimon.metrics.Histogram;
+import org.apache.paimon.metrics.groups.GenericMetricGroup;
+
+/** Metrics to measure a commit. */
+public class CommitMetrics {
+    private static final int HISTOGRAM_WINDOW_SIZE = 10_000;
+    protected static final String GROUP_NAME = "commit";
+
+    private final AbstractMetricGroup genericMetricGroup;
+
+    public CommitMetrics(String tableName) {
+        this.genericMetricGroup =
+                GenericMetricGroup.createGenericMetricGroup(tableName, GROUP_NAME);
+        registerGenericCommitMetrics();
+    }
+
+    public AbstractMetricGroup getMetricGroup() {
```

Comment:
nit: Add a `close` method in `CommitMetrics`. This can remind the user of `CommitMetrics` to close it. After that, add `@VisibleForTesting` to this method.